template.yaml

```diff
@@ -56,6 +56,29 @@ Resources:
           Properties:
             Schedule: cron(0 * ? * * *)
 
+
+  json2mvt:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      Handler: json2mvt.lambda_handler
+      Runtime: python3.7
+      Role: 'arn:aws:iam::958555546010:role/lambda-access-role'
+      CodeUri: functions/json2mvt/
+      Description: ''
+      Environment:
+        Variables:
+          DATA_SRC: !Select [1, !Split [":::", !GetAtt Bucket2.Arn]]
```

Comment:
this info is going to be in the lambda event...

Reply:
Am aware, however I want it in reference to vars I'll be pulling from

Reply:
yeah but this will always be the same as event["Records"][x]["s3"]["bucket"]["name"] so why not just use it from there

Reply:
Because it is no different to use environment variables. It's a choice of practice, one that makes more sense to me in terms of code readability and use. Just because the data is stored in the event message does not make it better.